crone/src/BufDiskWorker.cpp

```diff
@@ -437,6 +459,140 @@ noexcept {
     delete[] ioBuf;
 }
 
+void BufDiskWorker::mixBufferMono(const std::string &path, BufDesc &buf,
```

Comment:
i think these are close enough to `bufRead` counterparts to warrant combining them. id replace `bufReadMono` etc with these funcs and default params. (maybe still worth having 2 lua funcs, or make the extra params optional from lua too, by checking args count in `weaver`.)

Reply:
I ended up leaving only one function in lua, but made the extra parameters optional—which only needs work in `softcut.lua`, not in `weaver`.